.gitignore

```diff
@@ -21,15 +21,15 @@ coverage
 .lock-wscript
 
 # Compiled binary addons (http://nodejs.org/api/addons.html)
-build
+/build
 
 # Dependency directory
 # https://docs.npmjs.com/misc/faq#should-i-check-my-node-modules-folder-into-git
 node_modules
```

Comment:
I thought about that, but I don't think a directory name such as `node_modules` could ever be the source of confusion, plus this rule would cover us from accidental nested `node_modules` in other directories.